packages/client/index.html

```diff
@@ -40,69 +40,80 @@
       body,
       html {
         background-color: #f8f8f8;
+        margin: 0;
       }
-      #spinner {
+      .progress-container {
+        height: 100vh;
+        width: 100vw;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
```

Comment:
These styles could be inside the `div id="root"` or be named otherwise to not clash with anything the React app wants to render.